### PR TITLE
chore(apps/interface): set NavigationBarBottomLinksPopupBackground margin to 0

### PR DESCRIPTION
**Pull request by math-marcellino**
## RIS-561
1. Set margin to 0
2. update `data-testid` to `NavigationBarBottomLinksPopupBackground`

### CODE REVIEW

File: apps/interface/components/NavigationBarBottom/links.tsx [hunk 1]
```diff
@@ -280,12 +280,13 @@ export const NavigationBarBottomLinks = (props: ButtonProps) => {
...
                     {/* Backdrop blur */}
                     <Box
-                        data-testid="ChainSwitcherPopupBackground"
+                        data-testid="NavigationBarBottomLinksPopupBackground"
                         position="fixed"
                         width="100%"
                         height="100%"
                         left="0"
                         top="0"
+                        margin="0"
```
Comment (pyk) [inline apps/interface/components/NavigationBarBottom/links.tsx:289]: We need to add `!important` here.

```
margin: "0 !important";
```

**Review by pyk — CHANGES_REQUESTED** [review]
LGTM. T

File: apps/interface/components/NavigationBarBottom/links.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/NavigationBarBottom/links.tsx]: Updated sir, but what does `!important` mean(?)

**Review by pyk — APPROVED** [review]
LGTM. GG @math-marcellino

File: apps/interface/components/NavigationBarBottom/links.tsx [hunk 2]
```diff
@@ -280,12 +280,13 @@ export const NavigationBarBottomLinks = (props: ButtonProps) => {
                     </MenuList>
                     {/* Backdrop blur */}
                     <Box
-                        data-testid="ChainSwitcherPopupBackground"
+                        data-testid="NavigationBarBottomLinksPopupBackground"
```
Comment (pyk) [inline apps/interface/components/NavigationBarBottom/links.tsx:283]: nice catch

**Pull request merged**